data/json/items/fluff.json

```
@@ -239,6 +239,278 @@
     "flags": [ "NO_REPAIR" ],
     "use_action": [ "PLAY_GAME" ]
   },
+  {
+    "id": "senet",
+    "type": "GENERIC",
+    "category": "other",
+    "name": { "str": "senet set" },
+    "description": "A wooden box containing all the pieces needed to play a game of senet, and also a extremely small guide of how to play it.  Board game from ancient Egypt, it plays as modern chess with 3x10 grid and small amount of pieces.",
+    "weight": "800 g",
+    "volume": "2630 ml",
+    "price": 4300,
+    "price_postapoc": 5,
+    "material": [ "wood" ],
+    "symbol": "?",
+    "color": "brown",
+    "flags": [ "NO_REPAIR" ],
+    "use_action": [ "PLAY_GAME" ]
+  },
+  {
+    "id": "mahjong",
+    "type": "GENERIC",
+    "category": "other",
+    "name": { "str": "mahjong set" },
+    "description": "A very beautiful, and really huge plastic box, that contain all the pieces needed to play a game of mahjong, and also a extremely small, apparently poorly translated but comprehensible guide of how to play it.  Mahjong is a tile-based game where you need to find pairs of pieces and take the biggest score.",
+    "weight": "5530 g",
```

```suggestion
    "weight": "4000 g",
```
This was the last thing that gave a density error. Lowering the weight will fix this.